builtin/common/settings/dlg_settings.lua

```diff
@@ -616,6 +616,14 @@ local function get_formspec(dialogdata)
 
 		if show_reset then
 			local default = comp.setting.default
+			if comp.setting.type == "key" then
+				default = (default ~= "")
+					and core.get_key_description(default)
+
+					-- Indicates that the action does not have a corresponding keybinding.
```

Comment:
(Nitpick) the `~` in my previous suggestion was for indicating that the comment should be added as a translation note for Weblate (which is arguably why I suggested adding such a comment): https://github.com/luanti-org/luanti/blob/3b30e131ce141e7d1d357b6f15f56e80dcf9b3a5/util/updatepo.sh#L52-L53

```suggestion
					--~ Indicates that the action does not have a corresponding keybinding.
```

Reply:
Oh lol